proposals/log_output_v2.md

```diff
@@ -0,0 +1,337 @@
+# Proposal: Improve Jib Log Output
+
+## Motivation
+
+To have a more organized, readable, and aethetically-pleasing log output.
+
+## Goals
+
+- Consistency between Maven and Gradle
+- Organize across levels of verbosity
+
+## Proposed Design
+
+### Categories
+
+The log output will be organized into 4 categories:
+
+#### Lifecycle
+
+Messages that are part of the execution and will always show up.
+
+#### Info
+
+Messages that are extra information and do not need to be displayed by default. Note that for Maven, these will be at the same level as Debug.
+
+#### Debug
+
+Messages that are useful for debugging what went wrong, but not as useful (or convoluting) as user-facing information. These would be helpful to include in, for example, a bug report.
+
+#### Warn/Error
+
+These messages will be displayed by default, but as the warning/error format of the specific Maven/Gradle logger implementations. Jib should not explicitly log error messages, but rather throw those messages as part of exceptions.
+
+### Information to include
+
+The information to include, in the order in which they should appear are:
+
+1. (info) Where inputs are inferred from, including
+    1. The main class
+1. (info) Files that went into each layer
+    1. Classes, resources, dependencies
+1. (lifecycle/info) Important build steps
+    1. Show what’s skipped?
+    1. Asynchronous steps (the order may vary between runs)
+        - [lifecycle] Retrieving credentials for <base image>...
```

Comment:
FYI I think you need to escape these triangle brackets.

`<base image>` is showing up blank.